opentracing-mock/src/main/java/io/opentracing/mock/MockSpan.java

```diff
@@ -83,6 +86,13 @@ public long finishMicros() {
         return new ArrayList<>(this.logEntries);
     }
 
+    /**
+     * @return a copy of errors.
+     */
+    public List<RuntimeException> errors() {
```

Comment:
Agree name changed to `generatedErrors` and javadoc updated  :octocat: 

I was also thinking about `runtimeExceptions()`.. (naming dilemma..)